DSCResources/MSFT_xIPAddress/MSFT_xIPAddress.psm1

```diff
@@ -174,7 +179,5 @@ function ValidateProperties
     }
 }
 
-
-
-#  FUNCTIONS TO BE EXPORTED 
-Export-ModuleMember -function Get-TargetResource, Set-TargetResource, Test-TargetResource
+#  FUNCTIONS TO BE EXPORTED
+Export-ModuleMember -function Get-TargetResource, Set-TargetResource, Test-TargetResource
```

Comment:
Please end a file with a newline [See item 5 in style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)